epochX/cudacpp/CODEGEN/PLUGIN/CUDACPP_SA_OUTPUT/madgraph/iolibs/template_files/gpu/cudacpp.mk

```diff
@@ -833,7 +833,8 @@ endif
 
 $(testmain): $(GTESTLIBS)
 $(testmain): INCFLAGS +=  $(GTESTINC)
-$(testmain): LIBFLAGS += -L$(GTESTLIBDIR) -lgtest -lgtest_main
+$(testmain): LIBFLAGS += -L$(GTESTLIBDIR) -lgtest
+###$(testmain): LIBFLAGS += -lgtest_main # no longer necessary since we added main() to testxxx.cc
 
```

Comment:
@oliviermattelaer  need to run on my mac to check if it still compiles

Reply:
Ok please check on your Mac, I suggest we go ahead and in case we fix later. But I do not see why it should fail: it is taking LESS from gtest, not more (main is now hardcoded in mg5amc). Resolving